versioned_docs/version-4.4/building-extensions/custom-script/basic-script.md

```diff
@@ -8,15 +8,10 @@ sidebar_position: 2
 :::danger[Developer Notice]
 
 Creating a script which directly includes loads and boots the CMS framework is not recommended and only needed in
-rare cases. Depending on the usecase you should create a [console plugin](https://docs.joomla.org/J4.x:Writing_A_CLI_Application) 
-and extend the Joomla! console application. If you need to create a simple entry point for webrequests write an 
-[ajax plugin](https://docs.joomla.org/Using_Joomla_Ajax_Interface).
-
-:::
-
-:::note[TODO]
-
-Update the link to docs.joomla.org in the "Developer notice" above when an example has been added to this manual. 
+rare cases. 
+- If you plan to run the script from the server (in a cron job or from a terminal session) then you should use instead a [console plugin](../plugins/basic-console-plugin-helloworld.md). 
+- If you plan to allow people to run the script from a client browser then you should use an [ajax plugin](../plugins/ajax-plugin.md). 
+- If you wish to provide task functionality for an administrator, then using the Joomla Task Scheduler may be a possibility. 

```

Comment:
Not really happy with it.

* In the first point the cron job doesn't make sense in this case you should use point 3 (Task Scheduler).
* For point 2 I thin "can use" is better then "should" you should create your own component if you need to display a page with rendered html, ajax plugin should be used if you request data async or if you have no better choice.
* Not sure how the Task Scheduler can provide functionality for an administrator, it's only a cron job manager, so it doesn't really provid any buttons or interface to an user/admin.

Reply:
I've reverted the text in the custom script pages to what you specified, just changing the URLs to point to the joomla manual.